examples/plot_simulate_alpha.py

```diff
@@ -29,32 +29,28 @@
 print(params)
 
 ###############################################################################
-# Remove all evoked proximal feed parameters
-for key in params['*evprox*']:
-    del params[key]
-
-# Remove all evoked distal feed parameters
-for key in params['*evdist*']:
-    del params[key]
-
-###############################################################################
-# Now, we update a few rhythmic feed parameters
+# Update a few of the default parameters related to visualisation
 params.update({
     'dipole_scalefctr': 150000.0,
     'dipole_smooth_win': 0,
+    'sync_evinput': 1,  # XXX not applied (seed hack, see feed.py:_get_prng)
     'tstop': 310.0,
-    't0_input_dist': 50.0,
-    'tstop_input_dist': 1001.0,
-    'input_dist_A_weight_L2Pyr_ampa': 5.4e-5,
-    'input_dist_A_weight_L5Pyr_ampa': 5.4e-5,
-    'sync_evinput': 1,
-    "prng_seedcore_input_dist": 3
 })
 
 ###############################################################################
 # Now let's simulate the dipole and plot it
 net = Network(params)
-dpl = simulate_dipole(net)
+
+prng_initial_seed = 3 + 1  # XXX to match online docs
```

Comment:
```suggestion
prng_initial_seed = 4
```

in fact, you can even set it directly in the function below without specifying it as a separate variable. I don't want new users to be thinking too much about the seed mess. It's something we should first wrap our heads around. As long as it is consistent with the online docs, I think no further explanation is needed. I remember @rythorpe used similar tricks in the past so we have certainly deviated from the seeds used in HNN (GUI)